ButtonApp/app/src/main/java/com/android/sooz/simpleandroidapp/MainActivity.java

```diff
@@ -0,0 +1,95 @@
+package com.android.sooz.simpleandroidapp;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+
+public class MainActivity extends AppCompatActivity {
+    private int totalClicks;
+    private int aClicks;
+    private int bClicks;
+    private int cClicks;
+
+    private EditText mEditText;
+    private Button mManualUpdate;
+
+    private Button mClickMe;
+    private Button mAButton;
+    private Button mBButton;
+    private Button mCButton;
+
+    private TextView mClicks;
+    private TextView mATextView;
+    private TextView mBTextView;
+    private TextView mCTextView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        Log.d("clickapp","IT'S RUNNING!");
+
+        mEditText = findViewById(R.id.manualEntry);
+        mManualUpdate = findViewById(R.id.manualUpdate);
+
+        mClickMe = findViewById(R.id.clickMe);
+        mAButton = findViewById(R.id.aButton);
+        mBButton = findViewById(R.id.bButton);
+        mCButton = findViewById(R.id.cButton);
+
+        mClicks = findViewById(R.id.clicks);
+        mATextView = findViewById(R.id.aText);
+        mBTextView = findViewById(R.id.bText);
+        mCTextView = findViewById(R.id.cText);
+
+        mManualUpdate.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                String text = mEditText.getText().toString();
+
+               totalClicks = Integer.parseInt(text);
```

Comment:
There's actually a little bug here since the user isn't restricted to numeric input values. So If I type in a letter rather than a number, Integer.parseInt throws an exception. At some point I'm sure we'll cover how to restrict input fields to specific things. For now, probably the right approach is to wrap this in a try/catch.